packages/radix/src/components/ToggleButton.tsx

```diff
@@ -97,11 +97,11 @@ export const ToggleButton = React.forwardRef<HTMLButtonElement, ToggleButtonProp
                   borderColor: 'transparent',
                   backgroundColor: theme.colors.blue100,
                   boxShadow: `0 0 0 1px ${theme.colors.blue500}`,
-                  '&:not(:disabled):focus': {
+                  '&:focus': {
```

Comment:
These are not necessary anymore as we have changed all the specificity rules in primitives.